cmd/zc_processor.go

```diff
@@ -80,6 +83,30 @@ func (s *copyTransferProcessor) scheduleCopyTransfer(storedObject storedObject)
 		return nil // skip this one
 	}
 
+	if s.dryrunMode {
+		glcm.Dryrun(func(format common.OutputFormat) string {
+			if format == common.EOutputFormat.Json() {
+				jsonOutput, err := json.Marshal(copyTransfer)
+				common.PanicIfErr(err)
+				return string(jsonOutput)
+			} else {
+				// if remove then To() will equal to common.ELocation.Unknown()
+				if s.copyJobTemplate.FromTo.To() == common.ELocation.Unknown() { //remove
+					return fmt.Sprintf("DRYRUN: remove %v/%v",
+						s.copyJobTemplate.SourceRoot.Value,
+						srcRelativePath)
+				} else { //copy
+					return fmt.Sprintf("DRYRUN: copy %v/%v to %v/%v",
```

Comment:
Consider the case of / vs \ on Windows, do a test to make sure the printed path looks good. IIRC we already transform all the \ to / on Windows.

Reply:
Since this PR is focused on remove, I am deleting this piece of code for now because it is not currently being used. I will consider the different cases for / vs \ for copy and sync PR.